### DISCUSSION TITLE
stamp_duty versus adding a 'bollo a carico del cliente'

### DISCUSSION THREAD

**Discussion by ghost** [fattureincloud/api#308] [Q&A] [answered]
When creating an invoice, is it acceptable to set the "stamp_duty" to 2 instead of adding a 'bollo a carico del cliente' item?

[This page](https://developers.fattureincloud.it/docs/guides/invoice-creation/#5%EF%B8%8F%E2%83%A3-step-five-items-list)  shows how to add the item but it didn't work for me, the checkbox in the interface did not get checked. Setting the stamp_duty to 2 does work.

stamp_duty is not documented but I found it while retrieving an invoice.

**Comment by emajo** [comment, marked as answer]
Hi @EmondW, actually stamp_duty acts as 'bollo a carico del cliente' only if the document is not electronic, in electronic documents stamp_duty charges the 'bollo' to you, not to the customer, if you want to add the 'bollo a carico del cliente' in a einvoice you have to add the item as specified in the guide.

Thanks,
Emanuele.

**Comment by ghost** [comment]
Thank you Emanuele, for the explanation.

The effect you described (charging the 'bollo' to me) is what I wanted. Perhaps this should be added to the documentation?

**Reply by emajo** [reply]
Thanks @EmondW, we will add a better explanation of the bollo soon.
